### PR TITLE
split transpiler package

### PR DESCRIPTION
**Pull request by proppy**
That should help reviewing #8 

/cc @maruel (for some reason I can't assign you)

### CODE REVIEW

**Comment by maruel** [comment]
ah I didn't comment here because I had lgtm'ed over IM. You want me to click the merge button?

**Comment by proppy** [comment]
@lpereira might want to take a look too.

File: transpiler/transpiler.go [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+			fmt.Fprint(out, ";\n")
+		case *ast.AssignStmt:
+			if len(st.Lhs) > 1 {
+				return fmt.Errorf("unsupported # of lhs exprs: %v", st.Lhs)
+
+			}
+			if err := handleExpr(out, st.Lhs[0]); err != nil {
+				return fmt.Errorf("error handling left expr %v: %v", st.Lhs[0], err)
+			}
+			fmt.Fprintf(out, "=")
```
Comment (lpereira) [inline transpiler/transpiler.go:123]: `st.Tok` must be checked here; there are [many others](https://gist.github.com/lpereira/e9f0c8866adb89dd9855ad35fc3999e1#file-teste-go-L333-L374) besides simple assignment.

**Comment by lpereira** [comment]
LGTM as well.

**Pull request merged**